src/action.py

```diff
@@ -216,9 +238,12 @@ def make_actor(say):
     actor.add_keyword(_('repeat after me'),
                       RepeatAfterMe(say, _('repeat after me')))
 
-    # =========================================
-    # Makers! Add your own voice commands here.
-    # =========================================
+# =========================================
+# Makers! Add your own voice commands here.
+# =========================================
+
+    actor.add_keyword(_('power off'), PowerCommand(say, 'shutdown'))
+    actor.add_keyword(_('reboot'), PowerCommand(say, 'reboot'))
```

Comment:
ha! its funny you should ask. It works very well. I originally put "Pie" but the self conceded little board interpreted it as "Pi" (as displayed when  run manually via the main.py script) so I played along and took off the "e"

I did use a different command to accomplish this, bet here is my entry:

 actor.add_keyword(_('Pi power off'), SpeakShellCommandOutput(say, "sudo shutdown now",_('failed to shutdown')))

I also added reboot:

actor.add_keyword(_('Pi reboot'), SpeakShellCommandOutput(say, "sudo shutdown -r now", _('failed to reboot')))

Both commands are working as expected